rfcs/0074-community-coordination-hub.md

```diff
@@ -0,0 +1,153 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: Doron Behar <me@doronbehar.com>
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least). We don't have:
```

Comment:
I suggest defining "empowerment" and quoting or summarizing your definition of "leverage" rather than solely linking to a third-party website (which might disappear, be malicious, be taken over by spammers, etc.).